doc/cabal-package.rst

```diff
@@ -1454,24 +1435,25 @@ system-dependent values for these fields.
 
 .. pkg-field:: build-depends: library list
 
-    Declares the *library* dependencies required to build the current
-    package component; see :pkg-field:`build-tool-depends` for
-    declaring build-time *tool* dependencies. External library
-    dependencies should be annotated with a version constraint.
+    Declares the *library* dependencies (from the same package or other
+    packages) required to build the current package component. See
+    :pkg-field:`build-tool-depends` for declaring build-time *tool*
+    dependencies. Dependencies on libraries from another package should be
+    annotated with a version constraint.
 
     **Library Names**
 
-    External libraries are identified by the package's name they're
-    provided by, optionally followed by a colon and the library name
-    (available from ``cabal-version: 3.0``).
-    If the library name is absent, the main (unnamed) library will be used.
-    To refer to the main (unnamed) library explicitly, use the name of the
-    package (``foo:foo``).
-    Multiple libraries from the same package can be specified with the shorthand
-    syntax ``pkg:{lib1,lib2}```.
-
-    See section on :ref:`Multiple Public Libraries <publicsublibs>` for examples and more
-    information.
+    A library is identified by the name of the package that provides it,
+    optionally followed by a colon and the library's name (available from
+    ``cabal-version: 3.0``). If a library name is omitted, the package's 'main'
+    unnamed library will be used. To refer expressly to a package's 'main'
```

Comment:
```suggestion
    ``cabal-version: 3.0``). If a library name is omitted, the package's main
    library will be used. To refer expressly to a package's 'main'
```
As above.

Reply:
This is now reflected. The history of the Cabal specification has been moved out of the paragraph into a note.